Tasks/AzureFunctionAppV1/package.json

```diff
@@ -20,7 +20,10 @@
     "@types/mocha": "^5.2.7",
     "@types/node": "^10.17.0",
     "@types/q": "1.0.7",
-    "azure-pipelines-tasks-azurermdeploycommon": "3.207.0",
+    "azure-devops-node-api": "11.2.0",
+    "azure-pipelines-task-lib": "^4.1.0",
```

Comment:
There might be some incompatibility since it is a Node10 task. Please use `^3.4.0`